CHANGELOG.md

```diff
@@ -22,6 +24,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Initial release!
-
-[Unreleased]: https://github.com/thomaseizinger/create-pull-request/compare/1.0.0...HEAD
```

Comment:
As mentioned in the other PR, this needs to stay.

I'll update the master branch on my repo and then you can rebase / merge.